_pages/shuhaniii.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: shuhaniii
+pageUrl:https://shuhaniii.github.io/
```

Comment:
@shuhaniii remove the `https://` and `/`, and add a little spacing between the `:` and the URL